R/get.R

```diff
@@ -0,0 +1,597 @@
+#'    of rmd-static.}
+#'    \item{\strong{created_time}}{The timestamp (RFC3339) indicating when this 
+#'    content was created.}
+#'    \item{\strong{last_deployed_time}}{The timestamp (RFC3339) indicating when
+#'    this content last had a successful bundle deployment performed.}
+#'    \item{\strong{r_version}}{The version of the R interpreter associated 
+#'    with this content. The value null represents that an R interpreter is 
+#'    not used by this content or that the R package environment has not been 
+#'    successfully restored. Automatically assigned upon the successful 
+#'    deployment of a bundle.}
+#'    \item{\strong{py_version}}{The version of the Python interpreter 
+#'    associated with this content. The value null represents that a Python 
+#'    interpreter is not used by this content or that the Python package 
+#'    environment has not been successfully restored. Automatically assigned 
+#'    upon the successful deployment of a bundle.}
+#'    \item{\strong{build_status}}{}
+#'    \item{\strong{run_as}}{The UNIX user that executes this content. 
+#'    When null, the default Applications.RunAs is used. Applies 
+#'    only to executable content types - not static.}
+#'    \item{\strong{run_as_current_user}}{Indicates if this content is allowed 
+#'    to execute as the logged-in user when using PAM authentication. 
+#'    Applies only to executable content types - not static.}
+#'    \item{\strong{description}}{A rich description of this content}
+#'    \item{\strong{app_role}}{The relationship of the accessing user to this 
+#'    content. A value of owner is returned for the content owner. editor 
+#'    indicates a collaborator. The viewer value is given to users who are 
+#'    permitted to view the content. A none role is returned for 
+#'    administrators who cannot view the content but are permitted to view 
+#'    its configuration. Computed at the time of the request.}
+#'    \item{\strong{owner_first_name}}{The first name of the owner of the 
+#'    content.}
+#'    \item{\strong{owner_last_name}}{The last name of the owner of the 
+#'    content.}
+#'    \item{\strong{owner_username}}{The username of the owner of the 
+#'    content.}
+#'    \item{\strong{owner_guid}}{The unique identifier for the owner}
+#'    \item{\strong{owner_email}}{The email of the content owner}
+#'    \item{\strong{owner_locked}}{Is the owners user account locked?}
+#'    \item{\strong{is_scheduled}}{Is this content scheduled?}
+#'    \item{\strong{git}}{Is this content deployed via git or GitHub?}
+#' }
+#' 
+#' @details 
+#' Please see https://docs.rstudio.com/connect/api/#getContent for more information 
+#' 
+#' @examples
+#' \dontrun{
+#'   library(connectapi)
+#'   client <- connect()
+#'   
+#'   get_content(client, limit = 20)
+#' }
+#' 
+#' @export
+get_content <- function(src, filter = NULL, limit = 25, page_size = 25){
+  validate_R6_class("Connect", src)
+  
+  warn_experimental("'get_content'")
```

Comment:
```suggestion
  warn_experimental("get_content")
```

I don't think the extra quotes are needed. Just something I did for some weird reason on the `tbl_connect` endpoint 🤷‍♂